packages/theora/theora.0.4.1/opam

```diff
@@ -0,0 +1,38 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Bindings to libtheora"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0"
```

Comment:
Sorry I didn't catch this before:

I think this is not a big deal for now now, but be we are getting a linting error here because this a deprecated license identifier (see the last section of https://spdx.org/licenses/).

If you can switch to one of the current identifiers in your code base (e.g., `GPL-2.0-or-later` or `GPL-2.0-only`) here and/or in your code base, we won't hit this next time you do an update.

Reply:
This has been fixed, thanks!